priv/repo/migrations/20210416191647_add_session_stats_index.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Ret.Repo.Migrations.AddSessionStatsIndex do
+  use Ecto.Migration
+
+  @max_year 2030
+
+  def up do
+    for year <- 2018..@max_year,
+        month <- 0..11 do
```

Comment:
Is this written as `0..11` instead of `1..12` just to be consistent with the `CreateSessionStatsTable` migration?
https://github.com/mozilla/reticulum/blob/ca443c520f5a2cb9f0e5da1367b72bddedc8a38a/priv/repo/migrations/20180408231444_create_session_stats_table.exs#L31-L32

Reply:
Ah, yeah it was just a copy-paste. Could have simplified it, but I think this works.